app/models/proxy.rb

```diff
@@ -355,7 +355,7 @@ def save_and_deploy(attrs = {})
 
     return false unless saved
 
-    success = ProxyDeploymentService.call(self, v1_compatible: true)
+    success = ProxyDeploymentService.call(self)
```

Comment:
Shouldn't we remove this? What do you think? Can you explain why you keep it?

Reply:
Why keep the call or why keep `#save_and_deploy`?

Reply:
I think the Service is still useful, isn't it? To encapsulate the deployment logic, so removing the argument should be the right call.